src/components/Typography/index.stories.tsx

```diff
@@ -0,0 +1,43 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { typographyVariantList, typographyColorList, typopgrahyFontStyleList, typographyTagList } from './interfaces';
+
+import Typography, { ITypographyProps } from '.';
+
+export default {
+  title: 'Typography',
+  component: Typography,
+  argTypes: {
+    variant: {
+      control: {
+        type: 'select',
+      },
+      options: [undefined, ...typographyVariantList],
```

Comment:
Any reason why you kept the default values as undefined? maybe just set the first values of the list?
Particularly, I find it a bit ugly to land and see those undefined:
![image](https://user-images.githubusercontent.com/4645845/134731653-08b69602-211c-4eb2-9f6e-1a125b15595b.png)

Reply:
I wanted to set it to not pass the prop by default. I'll dig into the docs because I'm pretty sure there's a prettier way to do this!

Reply:
Ah I found that using the default value makes this prettier:

<img width="1786" alt="Screen Shot 2021-09-27 at 16 15 06" src="https://user-images.githubusercontent.com/5643276/134926119-6d0d920c-310d-4e04-8f20-c37fb66350aa.png">

Reply:
neat!